apps/server/src/api-data/rundown/rundown.service.ts

```diff
@@ -84,6 +84,7 @@ export async function editEntry(patch: PatchWithId): Promise<OntimeEntry> {
   }
 
   // we dont allow the user to change the cue to empty string
+  // FIXME this is not guarding as expected
```

Comment:
how come? can I make a PR to fix this?

Reply:
in the if statement on the line below it says
`isOntimeEvent(patch) && patch.cue === ''`
but the patch doesn't contain the event type so the check will always fail and therefor not prevent the edit